x/genutil/client/cli/init.go

```diff
@@ -179,6 +191,7 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 	cmd.Flags().String(flags.FlagChainID, "", "genesis file chain-id, if left blank will be randomly created")
 	cmd.Flags().String(FlagDefaultBondDenom, "", "genesis file default denomination, if left blank default value is 'stake'")
 	cmd.Flags().Int64(flags.FlagInitHeight, 1, "specify the initial block height at genesis")
+	cmd.Flags().String(FlagConsensusKeyAlgo, "ed25519", "algorithm to use for the consensus key")
```

Comment:
Is there a constant value defined somewhere that can be used?

Reply:
yes, there is but it isn't exported `https://github.com/hoangdv2429/cosmos-sdk/blob/6b25ca32eb209e7fde49d408eba9028a68082c5b/crypto/keys/ed25519/ed25519.go#L36`. Unless you want me to export it ?

Reply:
yes, I think it makes sense to export this